code/modules/research/techweb/all_nodes.dm

```diff
@@ -712,6 +712,16 @@
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
 	export_price = 5000
 
+/datum/techweb_node/electrostaff
+	id = "electrostaff"
+	display_name = "Riot Suppression Equipment"
+	description = "Speak softly and carry a massive stunbaton."
+	prereq_ids = list("adv_weaponry", "electric_weapons", "adv_bluespace")
+	design_ids = list("estaff")
```

Comment:
tbh I might place in Tele-riots in here well placing normal shields in adv weaponds
Or maybe add in basic stun stick as well